feature-libs/product-configurator/rulebased/components/attribute/types/checkbox/configurator-attribute-checkbox.component.ts

```diff
@@ -34,16 +40,7 @@ export class ConfiguratorAttributeCheckBoxComponent
    * Fired when a check box has been selected i.e. when a value has been set
    */
   onSelect(): void {
```

Comment:
can we remove this `onSelect()`, and in html all `onSelect(assembleSingleValue())` directly?

Reply:
True. Done!